### DISCUSSION TITLE
Has development of SumatraPDF Ended?

### DISCUSSION THREAD

**Discussion by jg-github** [sumatrapdfreader/sumatrapdf#3554] [General]
Greetings from Berlin,

As far as I'm able to see, the last SumatraPDF pre-release build version (15481) was built on 2023-04-18. Are any more coming?

Any relevant info will be appreciated.

Thanks.

Best regards,
Joe

**Comment by GitHubRulesOK** [comment]
Well the Daily runs are not needing anything to be uploaded at present, 
This project is  a single persons part time occupation and they are entitled to have a holiday between many other priorities like feeding family by a paid living. last activity on GitHub was 3 days ago after long break.

**Comment by jg-github** [comment]
I assume the second part of your assertion is accurate.As for the first part, l'm certainly not in a position to judge what's needed or not.All l can say is that there hasn't been a nightly since April. To me, it's a long time and a long 'summer vacation'.By the way, l'm in no hurry; just wondering.

**Comment by paradoxicallist** [comment]
I agree, it's been a long time since the last nightly build. I hope this pdf reader is not abandoned. Thanks for your great work contributors.
